tests/src/kernel/hipLaunchParm.cpp

```diff
@@ -577,16 +577,11 @@ __global__ void vAdd(float* a) {}
 // Some wrapper macro for testing:
 #define WRAP(...) __VA_ARGS__
 
-#include <sys/time.h>
 #define GPU_PRINT_TIME(cmd, elapsed, quiet)                         \
```

Comment:
Just a cosmetic change. But can we rename the MACRO to something else? Say MY_LAUNCH_MACRO or something? GPU_PRINT_TIME seems odd especially since we don't have any GPU timing code or prints in this macro.